supervision/detection/line_counter.py

```diff
@@ -81,6 +81,8 @@ def __init__(
         text_scale: float = 0.5,
         text_offset: float = 1.5,
         text_padding: int = 10,
+        custom_in_text: str = '',
```

Comment:
Could we make those arguments `Optional[str] = None`?

Reply:
Sure! It's great to learn that python allows this type of value 🧐. Today I learned something new about python

Reply:
I love to hear that